arboretum/permissions/fetchers/github/fetch_org_permissions.py

```diff
@@ -50,3 +50,33 @@ def fetch_repo_forks(self):
                             f'repos/{org}/{repo}/forks'
                         )
                     evidence.set_content(json.dumps(forks))
+
+    def fetch_repo_teams(self):
+        """Fetch Github repository teams."""
+        for config in self.config.get('org.permissions.org_integrity.orgs'):
+            host, org = config['url'].rsplit('/', 1)
+            url_hash = collabs.get_sha256_hash([config['url']], 10)
+            path = ['permissions', f'gh_teams_{url_hash}.json']
+            description = f'Teams in the {org} GH org'
```

Comment:
This description isn't completely accurate.  I believe that this evidence is repo access for teams in the org.  Maybe something like `f'Repo access for GH teams in the {org} GH org'` would be better?

Reply:
We don't have in plan to add more methods for the moment, we need to implement the check shortly so I think we can leave the things as they are. 
I have changed the description as suggested.